docs/mine/lotus/split-markets-miners.md

```diff
@@ -8,13 +8,39 @@ breadcrumb: 'Splitting main miner and markets service processes'
 
 {{ $frontmatter.description }}
 
-## Background
+[[TOC]]
+
+## Concepts
+
+Lotus performs mining operations, such as sealing files into sectors, calculating proofs over
+those files and submitting the proofs on chain.
+Lotus also performs markets operations, providing storage and serving retrievals to clients.
+
+It is now possible to run mining and markets operations in separate processes. Service providers can
```

Comment:
```suggestion
It is now possible to run mining and markets subsystems in separate processes. Service providers can
```

Reply:
```suggestion
It is possible to run mining and markets operations in separate processes. Service providers can
```

Woudl remove the temporality, since it'll be become outdated by the next release.